ktor-client/ktor-client-tests/build.gradle

```diff
@@ -19,8 +19,8 @@ class KtorTestServer extends DefaultTask {
             def main = mainClass.getMethod("startServer")
             server = main.invoke(null)
         } catch (cause) {
-            println("[TestServer] ERROR: $cause")
-            cause.printStackTrace()
+//            println("[TestServer] ERROR: $cause")
+//            cause.printStackTrace()
```

Comment:
?

Reply:
It prevents logging of address already in use(when testing locally)